src/components/Tricots.js

```diff
@@ -3,50 +3,65 @@ import {Header, List, Image} from "semantic-ui-react";
 import {Helmet} from "react-helmet";
 import SingleTrikot from "./SingleTrikot";
 import * as maillotActions from '../actions/maillotActions'
+import * as riderStageConnectionsActions from "../actions/riderStageConnectionsActions";
 import store from '../store'
 import {connect} from "react-redux";
+import {Route, Switch, Link} from 'react-router-dom';
+import {Menu} from 'semantic-ui-react';
+import TricotsStart from "./TricotsStart";
+import TricotsActual from "./TricotsActual";
 
 class Trikots extends Component {
   constructor(props){
     super(props);
 
     this.state = {
-      updated: false
+        activeItem : 'trikotsStart'
+
     }
   }
 
-  fetchCurrentMaillots(id) {
-    store.dispatch(maillotActions.getCurrentMaillots(id));
-    this.setState({updated: true});
-  }
+    handleMenuItemClick = (e, {name}) => this.setState({activeItem: name})
 
   render() {
         const {maillots} = this.props;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "maillots" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of the unused 'maillots' variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AUnusedVariable)